src/layouts/index.tsx

```diff
@@ -15,18 +17,18 @@ export const menuItems = [
 ];
 
 interface DefaultLayoutProps extends React.HTMLProps<HTMLDivElement> {
-  location: {
+  location?: {
     pathname: string;
   };
-  children: any;
+  children?: any;
 }
 
-export default class DefaultLayout extends React.PureComponent<DefaultLayoutProps, void> {
-  render() {
-    const { pathname } = this.props.location;
-    const isHome = pathname === "/";
+export default (props: DefaultLayoutProps): JSX.Element => {
```

Comment:
Some changes around typings make that TS is picky about class form components here.